backend/subroker/views.py

```diff
@@ -1,3 +1,392 @@
 from django.shortcuts import render
+from django.db.models.query import EmptyQuerySet
+from django.http import HttpResponse, HttpResponseNotAllowed, HttpResponseBadRequest, JsonResponse
+from django.contrib.auth.models import User
+from django.contrib.auth import login as auth_login, logout
+from django.views.decorators import csrf
+from django.views.decorators.csrf import ensure_csrf_cookie
+import json
+from json.decoder import JSONDecodeError
+from .models import Ott, Group, Content, Review
+from django.core.exceptions import ObjectDoesNotExist
+
+
+#signup/ : User Sign Up
+def signup(request):
+    if request.method == 'POST':
+        req_data = json.loads(request.body.decode())
+        username = req_data['username']
+        password = req_data['password']
+        User.objects.create_user(username, password)
```

Comment:
@muhwagwa I have tested this, and User.objects.create_user() has to be following order ```create_superuser(username, email=None, password=None, **extra_fields)```